robotoff/app/middleware.py

```diff
@@ -3,7 +3,7 @@
 
 class DBConnectionMiddleware:
     def process_resource(self, req, resp, resource, params):
-        db.connect()
+        db.connect(reuse_if_open=True)
```

Comment:
Nice.

Reply:
Yeah, it was necessary to be able to run the integration tests. Not sure why - but it seems the app starts up twice when I run them..?